api/auth/user/UserRestHandler.go

```diff
@@ -445,6 +447,55 @@ func (handler UserRestHandlerImpl) DeleteUser(w http.ResponseWriter, r *http.Req
 	common.WriteJsonResp(w, err, res, http.StatusOK)
 }
 
+func (handler UserRestHandlerImpl) BulkDeleteUsers(w http.ResponseWriter, r *http.Request) {
+	userId, err := handler.userService.GetLoggedInUser(r)
```

Comment:
We can get it from context

Reply:
Currently no user id is present in context. can be covered in phase -3

Reply:
Okay, will fix this in later phase